### ISSUE TITLE
Posting GeKrety log is broken

### ISSUE THREAD

**Issue by kumy** [cgeo/cgeo#5208]
Just found that posting GK seems broken. I don't think this is related to our c:geo code, but rather something wrong at GK.
The returned message, is not XML so it break the response parser.

The error extracted from the response is "No username supplied".
The parameters passed to the request seems good for me.

Can someone cross check before we ping Filips.
Thanks

**Comment by kumy** [comment]
```
0 = {BasicNameValuePair@5555} "secid=***hidden***"
1 = {BasicNameValuePair@5556} "gzip=0"
2 = {BasicNameValuePair@5557} "nr=***hidden***"
3 = {BasicNameValuePair@5558} "formname=ruchy"
4 = {BasicNameValuePair@5559} "logtype=5"
5 = {BasicNameValuePair@5560} "data=2015-09-27"
6 = {BasicNameValuePair@5561} "godzina=18"
7 = {BasicNameValuePair@5562} "minuta=38"
8 = {BasicNameValuePair@5563} "comment=Test cgeo"
9 = {BasicNameValuePair@5564} "app=c:geo"
10 = {BasicNameValuePair@5565} "app_ver=developer build"
11 = {BasicNameValuePair@5566} "mobile_lang=en_US.UTF-8"
```
https://gist.github.com/kumy/5530d307851e28b79889#file-badresponse-html

**Comment by kumy** [comment]
```
09-27 19:23:41.736  20480-20663/cgeo.geocaching E/cgeo﹕ [AsyncTask #5] GeokretyRuchyXmlParser: Error Parsing geokret
    org.xmlpull.v1.XmlPullParserException: unterminated entity ref (position:START_TAG <link href='stylesheet'>@1:541 in java.io.StringReader@1fb52f55)
            at org.kxml2.io.KXmlParser.readEntity(KXmlParser.java:1219)
            at org.kxml2.io.KXmlParser.readValue(KXmlParser.java:1401)
            at org.kxml2.io.KXmlParser.parseStartTag(KXmlParser.java:1122)
            at org.kxml2.io.KXmlParser.next(KXmlParser.java:372)
            at org.kxml2.io.KXmlParser.next(KXmlParser.java:313)
            at cgeo.geocaching.connector.trackable.GeokretyParser$GeokretyRuchyXmlParser.parse(GeokretyParser.java:287)
            at cgeo.geocaching.connector.trackable.GeokretyParser.parseResponse(GeokretyParser.java:321)
            at cgeo.geocaching.connector.trackable.GeokretyConnector.postLogTrackable(GeokretyConnector.java:285)
            at cgeo.geocaching.connector.trackable.GeokretyLoggingManager.postLog(GeokretyLoggingManager.java:33)
            at cgeo.geocaching.LogTrackableActivity$Poster.doInBackgroundInternal(LogTrackableActivity.java:427)
            at cgeo.geocaching.LogTrackableActivity$Poster.doInBackgroundInternal(LogTrackableActivity.java:413)
            at cgeo.geocaching.utils.AsyncTaskWithProgress.doInBackground(AsyncTaskWithProgress.java:105)
            at android.os.AsyncTask$2.call(AsyncTask.java:292)
            at java.util.concurrent.FutureTask.run(FutureTask.java:237)
            at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
            at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
            at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
            at java.lang.Thread.run(Thread.java:818)
```

**Comment by kumy** [comment]
Moreover, when the Exception is raised, no error is detected.

**Comment by kumy** [comment]
I've cross checked with my wife's phone and an old version.

@filipsPL does something changed in the API? Could you please check on you side?

**Comment by filipsPL** [comment]
tl;dr
should be fixed now.

We observed very large number of requests recently. Most of them was:  '"GET /export2.php?inventory=1&secid HTTP/1.1"', coming from various sources (there was no secid value set).  This, for some reasons, made the database reach its limits, so I've made a quick and dirty filter for the requests. Which has broken some code. But now it shoud be OK now.

**Comment by kumy** [comment]
@filipsPL Thanks!

**Comment by kumy** [comment]
Confirm fixed.
